### DISCUSSION TITLE
[Modding] AnyOutpost unusueable, "Outpost" location hardcoded

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#13485] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

When using "AnyOutpost" in a mission locationtype parameters, it is hardcoded to be anything that isn't empty, and isn't abandoned.

    <LocationType from="Outpost" to="AnyOutpost"/>

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/6c4f126f-c432-4657-9a33-669b40ffcf42)

**This means you cannot use "Anyoutpost" if you have nodes that arent empty but arent outposts either; such as special outpost types, or special nodes.** 

The game will trigger events in these functionally non-outpost nodes as if they were outposts. 

Suggest a parameter in Locationtypes that such as `IsAnyOutpost="false"`

Related to  https://github.com/FakeFishGames/Barotrauma/issues/7497 regarding unsuitable destination nodes (see my comment there)

### Reproduction steps

Any event with "anyoutpost" defined

`<EventSet identifier="exampleset" leveltype="outpost" locationtype="anyoutpost" ...`

### Version

v1.2.8.0 (Winter Update hotfix 2)

**Comment by AzureNocte** [comment]
I suppose it could be the reason some modded missions can not trigger in Dynamic Europa?
